Sources/RInAppMessaging/RInAppMessaging.swift

```diff
@@ -102,13 +94,12 @@ import RSDKUtils
                                                      router: router,
                                                      randomizer: randomizer,
                                                      displayPermissionService: displayPermissionService)
-            initializedModule?.aggregatedErrorHandler = { error in
-                errorDelegate?.inAppMessagingDidReturnError(error)
-            }
-            initializedModule?.delegate = delegate
+            initializedModule?.aggregatedErrorHandler = errorCallback
+            initializedModule?.onVerifyContext = onVerifyContext
             initializedModule?.initialize(deinitHandler: {
                 self.initializedModule = nil
                 self.dependencyManager = nil
+                self.onVerifyContext = nil
```

Comment:
You don't need to nil the `aggregatedErrorHandler` ?

Reply:
I was thinking about that. And I figured that setting static properties to nil when initialization is cancelled doesn't sound right. In fact I forgot to remove this line before pushing the changes. I wanted to leave `onVerifyContext` and `aggregatedErrorHandler` values in case of deinit event.

Reply:
Sure